test/lwc-dev-server/index.test.ts

```diff
@@ -40,10 +40,8 @@ describe('lwc-dev-server', () => {
   });
 
   beforeEach(async () => {
-    $$.SANDBOX.stub(ConfigUtils, 'getOrCreateIdentityToken').resolves('testIdentityToken');
```

Comment:
These were not needed and we forgot to delete/clean them up in the past PRs so just cleaning up now.